device-types/Ubiquiti/UP-AI-Port.yaml

```diff
@@ -0,0 +1,68 @@
+---
+manufacturer: Ubiquiti
+model: AI Port
+slug: ubiquiti-up-ai-port
+part_number: UP-AI-Port
+u_height: 0
+is_full_depth: false
+airflow: passive
+weight: 650
+weight_unit: g
+interfaces:
+  - name: eth0
+    label: PoE IN
+    type: 1000base-tx
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
```

Comment:
Looking at the spec sheet this is POE++ so would need to be `poe_type: type3-ieee802.3bt`